src/graph/graphcontext.c

```diff
@@ -243,9 +278,7 @@ void GraphContext_DeleteNodeFromIndices(GraphContext *gc, Node *n) {
 
 // Register a new GraphContext for module-level tracking
 void GraphContext_RegisterWithModule(GraphContext *gc) {
```

Comment:
This function is called from `GraphContextType_RdbLoad` which relied on the locking which this PR removed.

Reply:
restored